src/create-cluster.ts

```diff
@@ -48,16 +46,16 @@ export async function createCluster(
     startArgs.push('--mount-string', mountString);
   }
 
-  // update PATH to include minikube
-  env.PATH = getMinikubePath();
-
-  env.MINIKUBE_HOME = getMinikubeHome();
+  const env: Record<string, string> = {
+    PATH: getMinikubePath(),
+    MINIKUBE_HOME: getMinikubeHome() ?? '',
+  };
```

Comment:
It is removed, because it is not needed, podman desktop automatically adds it. See [main/src/plugin/util/exec.ts#L50](https://github.com/podman-desktop/podman-desktop/blob/1e432899a5f8321917c615e7c15de32a39de94a0/packages/main/src/plugin/util/exec.ts#L50)

Moreover process.env is incompatible with our RunOptions#env. 

`process.env` is typed as `ProcessEnv`, which is the following 
```
interface ProcessEnv extends Dict<string> {
                /**
                 * Can be used to change the default timezone at runtime
                 */
                TZ?: string;
            }
```

This is not compatible with our `RunOptions#env` defined as

```
export interface RunOptions {
    env?: { [key: string]: string };
...
```

Reply:
handled by https://github.com/podman-desktop/extension-minikube/pull/224